lib/config/recommended.js

```diff
@@ -91,4 +91,13 @@ export default {
     'style-concatenation': 'error',
     'table-groups': 'error',
   },
+  overrides: [
+    {
+      files: ['**/*.gjs', '**/*.gts'],
+      rules: {
+        'no-curly-component-invocation': 'off',
+        'no-implicit-this': 'off',
```

Comment:
Mind adding a comment next to each of these rules explaining why they are disabled for gjs/gts?

Reply:
updated!

Reply:
Lint failing.

Reply:
that's what I get for trying to do this all in-browser :upside_down_face: